src/lang-promql/complete/hybrid.ts

```diff
@@ -74,15 +76,15 @@ const autocompleteNode = {
 
 const snippets: readonly SnippetSpec[] = [
   {
-    keyword: 'sum(rate(<input vector>[5m]))',
+    keyword: 'sum(rate(&lt;input vector>[5m]))',
```

Comment:
I think you I preferred how you had these snippets non-escaped a few commits ago. If we always just escape it during the autocomplete, then when you define snippets here, you don't have to worry about HTML escaping (or can do it wrong and forget it).

Reply:
It's actually in order to avoid to match the string that replaces the double quote.
Because well if you are typing `sum` for example, then if everything is escaped, then the snippet `label_replace` came up.

Reply:
Well I think, the only way to have an accurate match will have to escape after the fuzzy. I don't have other solution to be sure that it won't match the escape characters :/

Reply:
Ok yeah, then let's leave it like this for now.

It's a bit of a pity that the fuzzy lib is unmaintained since 4 years, otherwise I would propose them to add an option to allow escaping for `ch` here: https://github.com/mattyork/fuzzy/blob/master/lib/fuzzy.js#L60

Maybe time for another fork sometime ;)